source/NVDAObjects/window/winword.py

```diff
@@ -912,8 +912,15 @@ def _normalizeFormatField(self,field,extraDetail=False):
 				# Translators: line spacing of at least x point
 				field['line-spacing']=pgettext('line spacing value',"at least %.1f pt")%float(lineSpacingVal)
 			elif lineSpacingRule==wdLineSpaceMultiple:
-				# Translators: line spacing of x lines
-				field['line-spacing']=pgettext('line spacing value',"%.1f lines")%(float(lineSpacingVal)/12.0)
+				multiLineSpacingVal = float(lineSpacingVal) / 12.0
+				
+				field['line-spacing'] = npgettext(
+					'line spacing value',
+					# Translators: line spacing of x lines
+					"%.1f line",
+					"%.1f lines",
+					multiLineSpacingVal,
+				) % (multiLineSpacingVal)

```

Comment:
These parentheses are unnecessary right? 

```suggestion
				) % multiLineSpacingVal
```